### PR TITLE
feat: add pt-br locale/update de terms

### PR DESCRIPTION
**Pull request by scottdover**
**Summary**
This adds new translations for pt-br (portuguese)

**Testing**
 - Made sure pt text was translated as expected

### CODE REVIEW

File: l10n/bundle.l10n.de.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "Cancelling job...": "Abbruch des Jobs...",
   "Cannot call self on ComputeSession with no id": "Kann self nicht für ComputeSessions ohne ID aufrufen",
   "Cannot call self on object with no id": "Kann self für ein Object ohne ID nicht aufrufen",
+  "Cannot find file: {file}": "Cannot find file: {file}",
```
Comment (scottdover) [inline l10n/bundle.l10n.de.json:7]: Hey @Criptic . Do you mind providing translations for these new terms when you have some time?

File: package.nls.zh-cn.json [hunk 2]
```diff
@@ -51,6 +51,8 @@
   "configuration.SAS.results.html.style": "指定 ODS HTML5 结果的样式。",
   "configuration.SAS.results.html.style.(auto)": "让扩展程序选择与颜色主题最匹配的样式。",
   "configuration.SAS.results.html.style.(server default)": "默认为 SAS 服务器上配置的样式。",
+  "configuration.SAS.results.sideBySide": "Display results to the side of the code",
```
Comment (scottdover) [inline package.nls.zh-cn.json:54]: Hey @scnwwu . Do you mind providing translations for these items when you have a moment?

Comment (scottdover) [reply package.nls.zh-cn.json:54]: Actually, nevermind for now. I'll create a separate PR for _just_ updating translations

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -85,3 +85,9 @@ Follow these steps to update a locale for the SAS Extension for VSCode:
 - Update any untranslated strings.
 - Verify your changes using `Launch Client`.
 - After you've verified changes, you can create a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) for review.
+
+### Locale contributors
+
+- German: David Weik
+- Chinese: Wei Wu
```
Comment (scnwwu) [inline CONTRIBUTING.md:92]: nit: Simplified Chinese, or how about including the ID as VS Code does?
https://github.com/microsoft/vscode-loc

**Review by scnwwu — APPROVED** [review]

**Comment by liu-lihua** [comment]
I checked the translated json file in master branch. I could see the Portuguese translations in it successfully.

**Pull request merged**